templates/c_main.html

```diff
@@ -5,16 +5,22 @@
     <title>Platform</title>
     <meta name="viewport" content=
         "width=device-width,initial-scale=1">
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/c_main.css') }}" />
+    <link rel="stylesheet" href="c_main.css">
   </head>
   
   <body>
       <div class="background">
         <div class="nav">
-          <a href="/quiz">Take Quiz</a>
-          <a href="/toolkit">View Toolkit</a>
-          <a href="/profile">Your Profile</a>
-          <a href="/home" class="active"><strong>Home</strong></a>
+          <div class="profile-dropdown-menu">
+            <a id="navmenu">Your Profile</a>
+            <ul class="profile-dropdown-content">
+              <a href=c_profile.html>Edit Profile</a>

```

Comment:
You cannot link another HTML file in a template. You have to specify a route you have defined in program1.py. Should be: href="/profile"